includes/EmailNotification.php

```diff
@@ -585,30 +585,28 @@ private function isArticleComment() {
 		// its not a blog.
 		return (
 			( $this->action === ArticleComment::LOG_ACTION_COMMENT ) &&
-			( $this->title->getNamespace() != NS_BLOG_ARTICLE )
+			( defined( 'NS_BLOG_ARTICLE' ) && $this->title->inNamespace( NS_BLOG_ARTICLE ) )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "NS_BLOG_ARTICLE" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)